libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -78,6 +78,19 @@ void CompilerUtils::toSizeAfterFreeMemoryPointer()
 	m_context << Instruction::SWAP1;
 }
 
+void CompilerUtils::revertWithStringData(Type const& _argumentType)
+{
```

Comment:
Maybe an assertion about the type of ``_argumentType``?

Reply:
`abiEncode` will complain if the conversion is not possible, but probably still a good idea.